fdbserver/MoveKeys.actor.cpp

```diff
@@ -1831,14 +1809,6 @@ ACTOR static Future<Void> startMoveShards(Database occ,
 						}
 					}
 
-					// Remove old dests from serverKeys.
```

Comment:
Why do we remove unassignServerKeys()?

Reply:
This is a followup cleanup, check line 1803 in the original file: `ASSERT(dest.empty());` There is no old dests due to the cleanUpDataMove() before each data move starts.